src/tex/appendix.tex

```diff
@@ -447,7 +447,20 @@ \subsection{Impact of sampling temperature}
 \end{figure}
 
 
+\clearpage
+\subsection{Summary of trends and recommendations}
+
+\oone model consistently leads across both \ChemBench corpus and \chembenchmini.  \ClaudeThreeFiveSonnet, \GPTFourO, \LlamaThreeOneFourZeroFiveBInstruct form the next tier of strong performers. These models show robust performance across all skills and difficulty levels. 
+Larger models generally perform better (\LlamaThreeOneFourZeroFiveBInstruct > \LlamaThreeOneSeventyBInstruct > \LlamaThreeOneEightBInstruct). However smaller but well-designed models like can compete with larger models. For example \GemmaTwoNineBIt has an overall accuracy of 0.47 on \ChemBench corpus, which is only 18\% less accurate than the larger \LlamaThreeOneEightBInstruct model.
+Clear progression in model families (e.g., \ClaudeThreeFiveSonnet >  \ClaudeThree > \ClaudeTwo). Newer versions consistently outperform their predecessors indicating improvements in scientific understanding of these model families.
+Basic tasks are consistently solved across all models and even top model show a drop in performance on the advanced tasks except for \oone model.
+Scores of knowledge intensive are generally lower than both Calculation and Reasoning. Reasoning tends to be the strongest skill among three.
 
+Based on these findings some scope for improving these models could be to focus on enhancing knowledge-based training. Possibly through improved pre-training on chemistry-specific texts and or with chemistry knowledge base integration. 
+Models show strongest performance in reasoning tasks, this suggests current training approaches are good at developing logical/reasoning capabilities. Larger models consistently perform better, However, there's evidence that well trained smaller models can compete with larger models. 
+Performance drops significantly for advanced tasks, this suggests including more advanced chemistry problems with complex, multi-step solutions in training or finetuning.
```

Comment:
```suggestion
\oone model consistently leads across both \ChemBench corpus and \chembenchmini.  \ClaudeThreeFiveSonnet, \GPTFourO, \LlamaThreeOneFourZeroFiveBInstruct form the next tier of strong performers. These models show robust performance across all skills and difficulty levels. 

Larger models generally perform better (\LlamaThreeOneFourZeroFiveBInstruct > \LlamaThreeOneSeventyBInstruct > \LlamaThreeOneEightBInstruct). However, smaller but well-designed models can compete with larger models. For example, \GemmaTwoNineBIt has an overall accuracy of 0.47 on \ChemBench corpus, which is only 18\% less accurate than the larger \LlamaThreeOneEightBInstruct model.
One can observe a clear progression of performance across model families (e.g., \ClaudeThreeFiveSonnet >  \ClaudeThree > \ClaudeTwo). Newer versions consistently outperform their predecessors, indicating improvements in scientific understanding of these model families.
Scores of knowledge-intensive are generally lower than those of Calculation and Reasoning. Reasoning tends to be the strongest skill among the three.

Based on these findings, some scope for improving these models could be to focus on enhancing knowledge-based training, possibly through improved pre-training on chemistry-specific texts or with chemistry knowledge-base integration. 
Models show the strongest performance in reasoning tasks, which suggests that current training approaches are good at developing logical/reasoning capabilities. Larger models consistently perform better. However, there's evidence that well-trained smaller models can compete with larger models. 
Performance drops significantly for advanced tasks, suggesting that more advanced chemistry problems with complex, multi-step solutions should be included in training or finetuning.
```

> which suggests that current training approaches are good at developing logical/reasoning capabilities.

or that our reasoning tasks are not very complex reasoning ;) 


> However, there's evidence that well-trained smaller models can compete with larger models.  

does this require a reference or do you mean Gemma?

Reply:
> or that our reasoning tasks are not very complex reasoning ;)

Mmm I agree. maybe we have a lot of simple reasoning tasks.  Should I change this to `which suggests that current training approaches are good at developing logical/reasoning capabilities required for solving tasks similar to our corpus` ?
or we can even consider removing recommendations all together.

Reply:
> does this require a reference or do you mean Gemma?

yes I meant Gemma.

Reply:
yeah i think `However, there's evidence that well-trained smaller models can compete with larger models.` is confusing maybe  I can rewrite this to maybe `Gemma and Phi demonstrates the potential for smaller models offering a compelling trade-off between performance and resource requirements.`